R/repl.R

```diff
@@ -202,3 +220,14 @@ py_repl <- function(
   }
   
 }
+
+#' Check Whether the Python REPL is Active
+#' 
+#' Check to see whether the Python REPL is active. This is primarily
+#' for use by R front-ends, which might want to toggle or affect
+#' the state of the Python REPL while it is running.
+#' 
+#' @export
```

Comment:
I think that we can make this non-exported for now (since R front ends can call non-exported methods without problem) just to not clutter up the API.